pkg/reconciler/ingress/ingress_shift_test.go

```diff
@@ -0,0 +1,345 @@
+package ingress
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	// Inject our fakes
+	fakesmmrclient "github.com/openshift-knative/knative-serving-networking-openshift/pkg/client/maistra/injection/client/fake"
+	fakerouteclient "github.com/openshift-knative/knative-serving-networking-openshift/pkg/client/openshift/injection/client/fake"
+	_ "knative.dev/pkg/client/injection/informers/istio/v1alpha3/gateway/fake"
+	_ "knative.dev/pkg/client/injection/informers/istio/v1alpha3/virtualservice/fake"
+	_ "knative.dev/pkg/client/injection/kube/informers/core/v1/endpoints/fake"
+	_ "knative.dev/pkg/client/injection/kube/informers/core/v1/pod/fake"
+	_ "knative.dev/pkg/client/injection/kube/informers/core/v1/secret/fake"
+	_ "knative.dev/pkg/client/injection/kube/informers/core/v1/service/fake"
+	_ "knative.dev/serving/pkg/client/injection/informers/networking/v1alpha1/ingress/fake"
+
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/sets"
+	clientgotesting "k8s.io/client-go/testing"
+
+	"knative.dev/pkg/configmap"
+	"knative.dev/pkg/controller"
+
+	_ "knative.dev/pkg/system/testing"
+	"knative.dev/serving/pkg/apis/networking/v1alpha1"
+	"knative.dev/serving/pkg/reconciler"
+	"knative.dev/serving/pkg/reconciler/ingress/resources"
+
+	. "knative.dev/pkg/reconciler/testing"
+	. "knative.dev/serving/pkg/reconciler/testing/v1alpha1"
+
+	_ "github.com/openshift-knative/knative-serving-networking-openshift/pkg/client/maistra/injection/informers/maistra/v1/servicemeshmemberroll/fake"
+	_ "github.com/openshift-knative/knative-serving-networking-openshift/pkg/client/openshift/injection/informers/route/v1/route/fake"
+	oresources "github.com/openshift-knative/knative-serving-networking-openshift/pkg/reconciler/ingress/resources"
+
+	maistrav1 "github.com/maistra/istio-operator/pkg/apis/maistra/v1"
+)
+
+// TestReconcileShift is an additional tests of TestReconcile in ingress_test.go.
+// It tests extra resources like SMMR, NetworkPolicy for networking-openshift.
+func TestReconcileShift(t *testing.T) {
```

Comment:
```suggestion
func TestReconcileOpenShift(t *testing.T) {
```

and rename the file to `ingress_openshift_test.go`?